### PR TITLE
#167729783 Testing Heroku Deployment

### PR DESCRIPTION
**Pull request by okafor-chidimma**
#### What does this PR do?

Creates an Updated Branch for Heroku deployment of staging app

#### Description of Task to be completed?
Prepares the staging app for deployment on Heroku 

#### How should this be manually tested?
checkout to this branch
run npm start in the terminal
Navigate to 

> https://shadowcat-backend-staging.herokuapp.com/

#### Any background context you want to provide?
Ensure that the repo is properly cloned locally

#### What are the relevant pivotal tracker stories?
#167729783

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "description": "Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web",
   "main": "index.js",
   "scripts": {
-    "build": "npx babel src --out-dir dist",
     "start": "npm run build && node ./dist/index.js",
+    "build": "npx babel src --out-dir dist",
```
Comment (davidfrank96) [inline package.json]: would you mind adding  `--copy-files` at the end of your build script so you won't have issues with swagger? but if u don't have that issue then you can procced

**Review by davidfrank96 — APPROVED** [review]
Looks good, nice work, just check out the comments if it works for you fine.

**Review by chibytez — APPROVED** [review]
nice work

File: package.json (hunk 1 above)
Comment (Stylll) [reply package.json]: @okafor-chidimma Do you need to do what @davidfrank96 recommended?

**Review by Stylll — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (okafor-chidimma) [reply package.json]: Okay...Let me do that

**Review by chibytez — APPROVED** [review]

**Review by okabamac — APPROVED** [review]
Looks good, but I don't know about running build anytime we want to start, else all look great

**Review by Stylll — APPROVED** [review]

**Pull request merged**